pom.xml

```diff
@@ -368,14 +368,14 @@
         <module>components/org.wso2.carbon.identity.api.user.common</module>
         <module>components/org.wso2.carbon.identity.api.user.association</module>
         <module>components/org.wso2.carbon.identity.api.user.challenge</module>
-        <module>components/org.wso2.carbon.identity.api.user.authorized.apps</module>
         <module>components/org.wso2.carbon.identity.api.user.approval</module>
         <module>components/org.wso2.carbon.identity.api.user.session</module>
         <module>components/org.wso2.carbon.identity.api.user.fido2</module>
         <module>components/org.wso2.carbon.identity.api.user.totp</module>
         <module>components/org.wso2.carbon.identity.api.user.application</module>
         <module>components/org.wso2.carbon.identity.api.user.recovery</module>
         <module>components/org.wso2.carbon.identity.api.user.functionality</module>
+        <module>components/org.wso2.carbon.identity.api.user.authorized.apps</module>
```

Comment:
why is this change required?

Reply:
Change reverted [81a8d70](https://github.com/wso2/identity-api-user/pull/129/commits/81a8d7074753b74f8bb43429db8d4ff80383675f)